locales-pending/limitations-apply-premium.ftl

```diff
@@ -2,8 +2,22 @@
 # License, v. 2.0. If a copy of the MPL was not distributed with this
 # file, You can obtain one at http://mozilla.org/MPL/2.0/.
 
-rules-and-restrictions-pill = Rules & Restrictions
 # Variables:
-# $discount_percentage_num is the amount discounted in percentage per month.
-# $discount_duration is the number of month(s) that users will pay the discounted price.
-limitations-apply-header = { $discount_percentage_num } off the next { $discount_duration } months
+# $discount_percentage_num is the amount discounted in percentage per month
```

Comment:
The variable and comment seem to imply that this does not include the `%` sign - and maybe it shouldn't? I wouldn't be surprised if there are locales in which it's common to add a space in between the number and the percentage, for example.

Same for percentages in the other strings.

Reply:
That's a valid point

```
            {l10n.getString("limitations-apply-title", {
              discount_percentage_num: "30%",
              discount_duration: 3,
            })}
```

Note that [JS provides ways to format that number](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Intl/NumberFormat/NumberFormat#options)